FLT/DedekindDomain/FiniteAdeleRing/BaseChange.lean

```diff
@@ -436,11 +436,46 @@ variable [Algebra K (FiniteAdeleRing B L)]
 noncomputable def FiniteAdeleRing.baseChange : FiniteAdeleRing A K →ₐ[K] FiniteAdeleRing B L where
   toFun ak := ⟨ProdAdicCompletions.baseChange A K L B ak.1,
     (ProdAdicCompletions.baseChange_isFiniteAdele_iff A K L B ak).1 ak.2⟩
-  map_one' := sorry
-  map_mul' := sorry
-  map_zero' := sorry
-  map_add' := sorry
-  commutes' := sorry
+  map_one' := by
+    refine ext B L ?_
```

Comment:
```suggestion
    ext
```
The lemma you're using is tagged `@[ext]` so you can just do this instead. Similar comment for `map_zero'`.